src/main/java/xbot/common/subsystems/vision/AprilTagVisionIOPhotonVision.java

```diff
@@ -43,7 +44,11 @@ public abstract static class FactoryImpl implements AprilTagVisionIOFactory {
     private static final int[] EMPTY_TAG_IDS_OBSERVATION = new int[0];
     private static final PoseObservation[] EMPTY_POSE_OBSERVATION = new PoseObservation[0];
     private static final TargetObservation[] EMPTY_TARGET_OBSERVATIONS = new TargetObservation[0];
-    private static final TargetObservation EMPTY_TARGET_OBSERVATION = new TargetObservation(0, new Rotation2d(), new Rotation2d(), new Transform3d(), 1);
+    private static final TargetObservation EMPTY_TARGET_OBSERVATION = new TargetObservation(0, 0, new Rotation2d(),
+            new Rotation2d(), new Transform3d(), 1, true);
+    // Using same heartbeat bounce as photonvision:
+    // https://github.com/PhotonVision/photonvision/blob/3c332db4bfe9083fc0311ae71cff92de588939ad/photon-lib/src/main/java/org/photonvision/PhotonCamera.java#L107
+    private static final double HEARTBEAT_DEBOUNCE_SEC = 0.5;
```

Comment:
maybe make this public, so the next time we need to reference it we don't need a copy-paste (I wish PV folks had made this a public constant)